Assignment1/styles.css

```diff
@@ -0,0 +1,292 @@
+body {
+  padding: 0px;
+  margin: 0px;
+  font: 100% Poppins, sans-serif;
+}
+
+#upper {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  z-index: 1;
+  color: #1b2021;
+  width: 1000px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+}
+
+nav {
```

Comment:
Try to not add styles to a native component. Instead add a class to that item and add the styles to that class.
Let's say you need 2 nav in your project but with different styling. If you are adding styling to `nav` element you would have the same styling for both elements and this is not what you want.